packages/next/pages/_document.tsx

```diff
@@ -547,7 +551,7 @@ export class NextScript extends Component<OriginProps> {
   render() {
     const {
       staticMarkup,
-      assetPrefix,
+      assetPrefix = '',
```

Comment:
```suggestion
      assetPrefix,
```

What case were you handling here? A separate PR would be better for this change.

Reply:
typescript compiler complains about variable can be undefined